### ISSUE TITLE
aria-label on TextFiled is given but its not recognizing it and Showing as Bug in accessibility test 

### ISSUE THREAD

**Issue by nansing90** [microsoft/fluentui#21975]
Environment Information
Package version(s): (fill this out)
Browser and OS versions: Every where
Lib :  '@fluentui/react/lib/TextField';

Actual behavior:
aria-label on TextFiled is given but its not recognizing it and Showing s Bug in the accessibility test 
![TextFiledArialabelIssue](https://user-images.githubusercontent.com/92431353/156981965-ce4777a7-fc9b-4ca7-9b12-65a2e74345c3.png)

Expected behavior:
Aria level should be detected and it shouldn't highlight  as accessibility issue. 
Priorities and help requested:
Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by nansing90** [comment]
Issue is resolved after removing         onRenderLabel={() => {
          return onRenderMargin(
            formatMessage({ id: LocalizationConstant.COLUMN_MARGIN }),
            sectionSettingsMargin.default,
            formatMessage({ id: LocalizationConstant.COLUMN_MARGIN_TOOLTIP })
          );
        }}
